src/test/run-pass/macros/macro-at-most-once-rep-2015.rs

```diff
@@ -0,0 +1,33 @@
+// run-pass
+#![allow(unused_mut)]
+// The logic for parsing Kleene operators in macros has a special case to disambiguate `?`.
+// Specifically, `$(pat)?` is the ZeroOrOne operator whereas `$(pat)?+` or `$(pat)?*` are the
+// ZeroOrMore and OneOrMore operators using `?` as a separator. These tests are intended to
+// exercise that logic in the macro parser.
+//
+// Moreover, we also throw in some tests for using a separator with `?`, which is meaningless but
+// included for consistency with `+` and `*`.
```

Comment:
This is also not true. Using a separator with `?` will result in a parsing error.

Reply:
It looks like I just forgot to update the comments the last time I updated this file. The test itself looks correct to me...